presto-docs/src/main/sphinx/release/release-326.rst

```diff
@@ -0,0 +1,48 @@
+=========================
+Release 326 (25 Nov 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix performance regression in joins. (:issue:`2047`)
+* Fix bug when query was cancelled during semantic analysis but semantic analysis time
+  counter was still counting time. (:issue:`2055`)
+* Add :doc:`/connector/memsql`. (:issue:`1906`)
+* Improve performance of ``INSERT`` and ``CREATE TABLE ... AS`` queries containing redundant
+  ``ORDER BY`` clauses. (:issue:`2044`)
+* Improve performance of processing map columns. (:issue:`2015`)
+
+Security Changes
+----------------
+
+* Deprecate Kerberos in favor of JWT for :doc:`/security/internal-communication`. (:issue:`2032`)
+
+Hive Changes
+------------
+
+* Fix an error when creating a table with S3 location when using ``file`` metastore. (:issue:`1664`)
+* Fix a CDH 5.x metastore compatibility issue resulting in the metastore discarding stats computed
+  with :doc:`/sql/analyze`. (:issue:`973`)
+* Improve performance for Glue metastore by fetching partitions in parallel. (:issue:`1465`)
+* Optimizations in ``sql-standard`` Hive security. (:issue:`1922`, :issue:`1929`)
```

Comment:
@kokosing, what is this item about? What, specifically, is being optimized?

Reply:
The number of calls to HMS is much lower to get user roles and table privileges. It affects a case where user has several roles.